applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -89,7 +89,7 @@
                                                 <meta itemprop="thumbnail" content="@Html(video.content.signedArticleImage)" />
                                                 <meta itemprop="thumbnailUrl" content="@Html(video.content.signedArticleImage)" />
 
-                                              @fragments.atoms.media(mediaAtom)
+                                              @fragments.atoms.media(mediaAtom, displayCaption = false)
```

Comment:
No, I wanted to override the setting here because we don't want to show the caption on the video standalone page.